HACKING.md

```diff
@@ -102,27 +98,100 @@ see the arXiv importers in `arxiv.py`. To add a new importer
 
 The importer is then registered with `papis` by adding it to `setup.py`. In the
 `entry_points` argument under `"papis.importer"` add
+
 ```
 myimporter=papis.myservice:Importer
 ```
+
 or see the existing examples.
 
-Adding Downloaders
-------------------
+## Adding Downloaders
 
 The difference between a downloader and an importer in `papis` is largely
 semantic. Downloaders are mostly meant to scrape websites or download files
-from a remote location. They can be implemented in a very similar way:
-
-- Create a class that inherits from `papis.downloaders.Downloader`.
-- Implement the `Downloader.match` method, which generally checks if a given
-  URI matches a website URL.
-- (Optional) Implement the `Downloader.get_data` method, which returns a dictionary.
-- (Optional) Implement the `Downloader.get_bibtex_url` method to return an URL
-  from which a BibTeX file can be downloaded. This is then parsed and merged
-  automatically.
-- (Optional) Implement the `Downloader.get_document_url` method to return an
-  URL from which a document (e.g. PDF file) can be downloaded.
-
-The downloader can then be added to the `"papis.downloader"` key in `setup.py`,
-similarly to an importer.
+from a remote location. They can be implemented in a similar way.
+
+As a example we implement a custom downloader for the [ACL Anthology](https://aclanthology.org/).
```

Comment:
The example you added here is great! However it reads more like a tutorial, so maybe it should be in the docs instead?

The docs for the plugin architecture are pretty nonexistent at the moment
https://github.com/papis/papis/blob/2fe163de082b883f804b19ed76d756b02f05dfdb/doc/source/plugins.rst
but we can start something in there.